src/store/create-store-module.js

```diff
@@ -84,6 +85,9 @@ export default (oidcSettings, storeSettings = {}, oidcEventListeners = {}) => {
     oidcAccessTokenExp: (state) => {
       return tokenExp(state.access_token)
     },
+    oidcScopes: (state) => {
```

Comment:
actually we don't need getters for only retrieving the state, but I tried to follow your code

Reply:
Nice. I've opted to use getters more for the future, if some middle layer features would need to be applied. Thanks for sticking with that!